Content.Shared/Xenoarchaeology/Equipment/SharedArtifactCrusherSystem.cs

```diff
@@ -33,6 +38,23 @@ private void OnStorageAfterOpen(Entity<ArtifactCrusherComponent> ent, ref Storag
         ContainerSystem.EmptyContainer(ent.Comp.OutputContainer);
     }
 
+    private void OnEmagged(Entity<ArtifactCrusherComponent> ent, ref GotEmaggedEvent args)
+    {
+        ent.Comp.AutoLock = true;
+        args.Handled = true;
+    }
+
+    private void OnStorageOpenAttempt(Entity<ArtifactCrusherComponent> ent, ref StorageOpenAttemptEvent args)
+    {
+        if (ent.Comp.AutoLock && ent.Comp.Crushing)
+            args.Cancelled = true;
+    }
+
+    private void OnExamine(Entity<ArtifactCrusherComponent> ent, ref ExaminedEvent args)
+    {
+        args.PushMarkup(ent.Comp.AutoLock ? Loc.GetString("artifact-crusher-examine-autolocks") : Loc.GetString("artifact-crusher-examine-no-autolocks"));
```

Comment:
these locale strings aren't in the commits. Did you forget to add/commit them before you made the PR?

Reply:
Forgot to add the file to git, committed it now.